integration-cli/docker_cli_inspect_test.go

```diff
@@ -460,6 +460,10 @@ func (s *DockerSuite) TestInspectUnknownObject(c *check.C) {
 }
 
 func (s *DockerSuite) TestInspectInvalidReference(c *check.C) {
+	// fixme: this test case is not implemented properly nor the engine
+	// handling the error properly.
+	// disable the test for now while waiting to be fixed.
+	c.Skip("skipping test case with flaw")
```

Comment:
What is the issue here?

Reply:
@tiborvass from `17.06.2-ee-11` engine, it's not returning the complete error message. 
```
[]
Error: No such object: FooBar
```
vs
```
[]
Error response from daemon: no such image: FooBar: invalid reference format: repository name must be lowercase
```
Plus this test case seems to be a duplicate from `TestInspectUnknownObject` above. After discussing with @thaJeztah, we decide to disable it for now.

Reply:
How could those two tests pass on any version of docker ? Aren't they mutually exclusive?

Reply:
From the other pull request https://github.com/moby/moby/pull/37003#discussion_r187942411 :

This test looks flawed; it doesn't test for the right part of the error message, which is at the end:

```
Error response from daemon: no such image: FooBar: invalid reference format: repository name must be lowercase
```

But there's actually a bug: `docker inspect` stops after this, and because of that masks other, valid, objects:

```
docker volume create FooBar
FooBar

docker inspect FooBar
[]
Error response from daemon: no such image: FooBar: invalid reference format: repository name must be lowercase

docker run -dit --name FooBar busybox
docker inspect FooBar 

# (outputs container inspect)
```

What's needed is;

- Test should check for `invalid reference format: repository name must be lowercase`
- Test should do `docker image inspect` (i.e. this error is specific to inspecting images)
- (not for this PR); we should fix `docker inspect` stopping at the first error it encounters to prevent it from masking valid objects